Python/simple_utils.py

```
@@ -0,0 +1,11 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text):
+  """Reverses the characters in a string."""
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)